### PR TITLE
ci: add deps and ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by Lutra-Fs**

### CODE REVIEW

**Comment by Lutra-Fs** [comment]
Need some test on pip. pip should correctly recognise and use pyproject.toml @Track07-cda

**Comment by Lutra-Fs** [comment]
ref https://stackoverflow.com/questions/62408719/download-dependencies-declared-in-pyproject-toml-using-pip

**Comment by Lutra-Fs** [comment]
Test on pip 22 in python 3.11 looks like we will need to limit the python version to <3.11

**Comment by ShaneHe023** [comment]
> Test on pip 22 in python 3.11 looks like we will need to limit the python version to <3.11

Do you have error messages or equivalent? BTW does pip v23 work with python 3.11?

**Review by ShaneHe023 — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+authors = [
+    {name = "techlancher-mlai-edge-physics", email = "example@example.com"},
+]
+dependencies = [
+    "phiflow>=2.4.0",
+    "tensorflow>=2.13.0",
+    "tensorflowjs>=4.10.0",
+    "onnx>=1.14.0",
+]
+requires-python = ">=3.9,<3.12"
```
Comment (Lutra-Fs) [inline pyproject.toml]:
```suggestion
requires-python = ">=3.9,<3.11"
```

**Review by ShaneHe023 — APPROVED** [review]

**Review by LorenzoValentine — APPROVED** [review]

**Pull request merged**